api/controllers/game/ready.js

```diff
@@ -1,55 +1,52 @@
-const gameAPI = sails.hooks['customgamehook'];
 const userAPI = sails.hooks['customuserhook'];
 
-module.exports = function (req, res) {
-  if (req.session.game && req.session.usr) {
```

Comment:
I've removed this check from the function body to replace it with the `isInGame` and `isLoggedIn` policies, which we generally do for other endpoints and which lets us have this endpoint focus on the business logic